rust/src/offer.rs

```diff
@@ -13,29 +10,32 @@ pub struct Offer {
     pub index: f64,
 }
 
-pub async fn get_offer() -> Result<Option<Offer>> {
+pub async fn get_offer() -> Option<Offer> {
     let client = reqwest::Client::builder()
         .timeout(crate::config::TCP_TIMEOUT)
-        .build()?;
+        .build()
+        .expect("reqwest client to build");
```

Comment:
❓ So this will never panic?

Reply:
I guess? looks odd to me that a builder should fail, but I am not sure.

Reply:
https://github.com/itchysats/10101/pull/550/commits/a0ea0da977c31ccd59c1a12da7f58b684100b1bc